devscripts/make_lazy_extractors.py

```diff
@@ -34,18 +33,14 @@ class {name}({bases}):
 
 
 def main():
-    lazy_extractors_filename = get_filename_args(default_outfile='yt_dlp/extractor/lazy_extractors.py')
-    if os.path.exists(lazy_extractors_filename):
-        os.remove(lazy_extractors_filename)
+    os.environ['YTDLP_NO_PLUGINS'] = 'true'
+    os.environ['YTDLP_NO_LAZY_EXTRACTORS'] = 'true'
```

Comment:
`'1'` would match our documentation for `YTDLP_NO_LAZY_EXTRACTORS` but it doesn't really matter
```suggestion
    os.environ['YTDLP_NO_PLUGINS'] = '1'
    os.environ['YTDLP_NO_LAZY_EXTRACTORS'] = '1'
```

Reply:
I'd rather change the documentation to reflect the implementation, so just say `nonempty` for `YTDLP_NO_LAZY_EXTRACTORS`